lib/geo.ex

```diff
@@ -118,4 +118,27 @@ defmodule Geo do
       end
     end
   end
+
+  defimpl Jason.Encoder,
```

Comment:
this adds a hard dependency on Jason (when a project doesn't happen to have it installed, compiling geo will fail) so I'd either:

Option 1: Add it as hard dependency:

```diff
- {:jason, "~> 1.2", only: :test},
+ {:jason, "~> 1.2"},
```

Option 2:

Guard this with an if:

```elixir
if Code.ensure_loaded?(Jason.Encoder) do
  defimpl Jason.Encoder, ...
end
```

Reply:
Hi, @wojtekmach! Good point, thanks for bringing this to my attention. I'll make sure to provide relevant changes A.S.A.P.

Appreciate your feedback.